website/frontend/templates/downloads.html

```diff
@@ -141,6 +141,12 @@
                 </thead>
                 <tbody>
                   <tr>
+                    <td>{{ _("All") }} (Flatpak)</td>
```

Comment:
Unfortunately, Python doesn't support gettext contexts: http://bugs.python.org/issue2504

I used "All distributions" in the latest push.